[0m14:45:32.571204 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f5f8c94cac0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f5f8b096230>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f5f8b0949a0>]}


============================== 14:45:32.575329 | a2d454be-9c64-425b-94fa-bc4f2eb7f18b ==============================
[0m14:45:32.575329 [info ] [MainThread]: Running with dbt=1.8.4
[0m14:45:32.576948 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'profiles_dir': '/root/.dbt', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt init taxis_chicago', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m14:45:32.589122 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse dict {'dir': PosixPath('/root/.dbt')}
[0m14:45:32.590333 [info ] [MainThread]: Creating dbt configuration folder at 
[0m14:45:32.592235 [debug] [MainThread]: Starter project path: /usr/local/lib/python3.10/site-packages/dbt/include/starter_project
[0m14:45:32.690032 [info ] [MainThread]: 
Your new dbt project "taxis_chicago" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m14:45:32.691178 [info ] [MainThread]: Setting up your profile.
[0m14:45:35.709566 [info ] [MainThread]: Profile taxis_chicago written to /root/.dbt/profiles.yml using target's sample configuration. Once updated, you'll be able to start developing with dbt.
[0m14:45:35.712715 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 3.2045212, "process_user_time": 1.234043, "process_kernel_time": 0.255949, "process_mem_max_rss": "121748", "process_in_blocks": "123128", "process_out_blocks": "344"}
[0m14:45:35.713763 [debug] [MainThread]: Command `dbt init` succeeded at 14:45:35.713663 after 3.21 seconds
[0m14:45:35.714428 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f5f8c94cac0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f5f8b097a00>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f5f8b0472b0>]}
[0m14:45:35.715237 [debug] [MainThread]: Flushing usage events
[0m15:14:49.720998 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f04a4d4cb20>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f04a3257a90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f04a3257b80>]}


============================== 15:14:49.725497 | 58683f75-fa63-4c24-a236-dd8d1b05bffc ==============================
[0m15:14:49.725497 [info ] [MainThread]: Running with dbt=1.8.4
[0m15:14:49.726977 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'profiles_dir': '/root/.dbt', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'False', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'log_format': 'default', 'invocation_command': 'dbt run', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m15:14:49.727976 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /etl_project/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m15:14:49.729445 [debug] [MainThread]: Resource report: {"command_name": "run", "command_wall_clock_time": 0.07896601, "process_user_time": 1.226013, "process_kernel_time": 0.118646, "process_mem_max_rss": "89524", "process_in_blocks": "68456", "process_out_blocks": "8", "command_success": false}
[0m15:14:49.730156 [debug] [MainThread]: Command `dbt run` failed at 15:14:49.730072 after 0.08 seconds
[0m15:14:49.730708 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f04a4d4cb20>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f04a3257af0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f04a3257b80>]}
[0m15:14:49.731421 [debug] [MainThread]: Flushing usage events
[0m15:29:36.003443 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f0db1d90a60>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f0db0255630>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f0db02555d0>]}


============================== 15:29:36.008795 | bfffc72c-3ba4-4876-918e-d462349d85b5 ==============================
[0m15:29:36.008795 [info ] [MainThread]: Running with dbt=1.8.4
[0m15:29:36.010010 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'debug': 'False', 'profiles_dir': '/root/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'False', 'log_format': 'default', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'target_path': 'None', 'invocation_command': 'dbt run', 'send_anonymous_usage_stats': 'True'}
[0m15:29:36.011456 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /etl_project/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m15:29:36.013271 [debug] [MainThread]: Resource report: {"command_name": "run", "command_wall_clock_time": 0.08911281, "process_user_time": 1.27043, "process_kernel_time": 0.040653, "process_mem_max_rss": "89896", "process_in_blocks": "26568", "command_success": false, "process_out_blocks": "0"}
[0m15:29:36.014150 [debug] [MainThread]: Command `dbt run` failed at 15:29:36.014052 after 0.09 seconds
[0m15:29:36.014834 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f0db1d90a60>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f0db02556c0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f0db02555d0>]}
[0m15:29:36.015702 [debug] [MainThread]: Flushing usage events
[0m15:33:38.465380 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f7578234b50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f7576665720>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f75766656c0>]}


============================== 15:33:38.470865 | 6aa42cb2-67fb-48f7-84f6-0668af0ed2c9 ==============================
[0m15:33:38.470865 [info ] [MainThread]: Running with dbt=1.8.4
[0m15:33:38.472073 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/root/.dbt', 'version_check': 'True', 'warn_error': 'None', 'log_path': 'logs', 'fail_fast': 'False', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'False', 'log_format': 'default', 'static_parser': 'True', 'invocation_command': 'dbt run', 'introspect': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m15:33:38.473377 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /etl_project/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m15:33:38.475265 [debug] [MainThread]: Resource report: {"command_name": "run", "command_wall_clock_time": 0.079544745, "process_user_time": 1.253628, "process_kernel_time": 0.079595, "process_mem_max_rss": "89800", "process_in_blocks": "26568", "command_success": false, "process_out_blocks": "0"}
[0m15:33:38.476067 [debug] [MainThread]: Command `dbt run` failed at 15:33:38.475971 after 0.08 seconds
[0m15:33:38.476846 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f7578234b50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f75766657b0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f75766656c0>]}
[0m15:33:38.477770 [debug] [MainThread]: Flushing usage events
